docs/explanations/about-sqlmodel.md

```diff
@@ -0,0 +1,44 @@
+# About SQLModel
+
+As the website of [SQLModel](https://sqlmodel.tiangolo.com/) mentions:
+
+!!! quote
+
+    `SQLModel`, SQL databases in Python, designed for simplicity, compatibility, and
+    robustness.
+
+## How and why do we use SQLModel
+
+`SQLModel` is used to create a simple and robust ORM for SQL databases. It is
+used to create models and interact with the database. It is a framework based on
+[Pydantic](https://pydantic.dev/) and [SQLAlchemy](https://www.sqlalchemy.org/).
```

Comment:
As Pydantic and SQLAlchemy are greatly used under the hood in this project, I would create distinctive explanations for them as well.